pages/style/overview.md

```diff
@@ -0,0 +1,99 @@
+---
+permalink: /style/
+layout: styleguide
+title: Style
+category: Style
+lead: We build System components from consistent global palettes of type scale, spacing units, and color. Your project’s custom theme palette provides a subset of the global palettes tailored to your project’s needs.
+type: docs
+---
+
+## The benefits of global style palettes
+
+<p>Our global palettes of <a href="{{ site.baseurl }}/style/type-scale/">type scale</a>, <a href="{{ site.baseurl }}/style/spacing-units/">spacing units</a>, and <a href="{{ site.baseurl }}/style/color/">color</a> are the visual building blocks any component. When we use these palettes consistently, not only do individual components fit better within the design of the larger website, but USWDS websites have better coherence across the entire federal government. This coherence provides a more familiar, reliable user experience.</p>
```

Comment:
Should this be in markdown like the following paragraphs?

Reply:
Will the `<a>` links work in a markdown paragraph?

Reply:
You can use the link markdown format and it will work `[]()`

Reply:
OK, cool. Dunno why I thought that wouldn't work...
One sec.